feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreen.kt

```diff
@@ -9,10 +14,63 @@ import com.epmedu.animeal.navigation.navigator.LocalNavigator
 @Composable
 fun AboutScreen() {
     val navigator = LocalNavigator.currentOrThrow
+    val context = LocalContext.current
+    val facebookWebIntent = remember {
+        Intent(Intent.ACTION_VIEW, Uri.parse("https://www.facebook.com/animalprojectgeorgia/"))
+    }
+    val facebookAppIntent =
+        remember { Intent(Intent.ACTION_VIEW, Uri.parse("fb://page/" + "100226375359696")) }
+    val instagramAppIntent = remember {
+        Intent(
+            Intent.ACTION_VIEW,
+            Uri.parse("https://www.instagram.com/animalproject.ge/")
+        )
+    }.setPackage("com.instagram.android")
+    val instagramWebIntent = remember {
+        Intent(
+            Intent.ACTION_VIEW,
+            Uri.parse("https://www.instagram.com/animalproject.ge/")
+        )
+    }
+    val linkedinWebIntent = remember {
+        Intent(
+            Intent.ACTION_VIEW,
+            Uri.parse("https://www.linkedin.com/company/animalprojectgeorgia/")
+        )
+    }
+//    TODO: update webIntent url when ready
+    val webIntent = remember {
+        Intent(
+            Intent.ACTION_VIEW,
+            Uri.parse("https://www.linkedin.com/company/animalprojectgeorgia/")
+        )
+    }
 
     BottomBarVisibility(HIDDEN)
 
     AboutScreenUI(
         onBack = navigator::popBackStack,
+        onSocialFacebookClick = {
+            try {
+                context.startActivity(facebookAppIntent)
+            } catch (e: ActivityNotFoundException) {
+                e.printStackTrace()
+                context.startActivity(facebookWebIntent)
+            }
+        },
+        onSocialInstagramClick = {
+            try {
```

Comment:
Don't repeat yourself. Extract the try-catch block to extension function `Context.startAppOrBrowser(
    appIntent: Intent,
    webIntent: Intent
)` in `library/extensions/Context`